interface/main/messages/messages.php

```diff
@@ -23,614 +24,686 @@
+    <meta name="author" content="OpenEMR: MedExBank">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <style>
+      label { font-weight:400;}
+      </style>
+  </head>
+
+  <body class="body_top" style="font-size:0.8em;">
+    <div class="row">
+        <?php
+        if (empty($_REQUEST['nomenu'])) $MedEx->display->navigation($logged_in); ?>
+    </div>
+    <?php if (!empty($_GET['go'])) { ?>
+            <?php
+            if (($_REQUEST['go']=="setup")&&(!$logged_in)) {
+                $stage = $_REQUEST['stage'];
+                if (!is_numeric($stage)) {
+                    echo "<br /><br /><span class='title'>$stage ".xlt('Warning: This is not a valid request').".</span>";
+                } else{
+                    $MedEx->setup->MedExBank($stage);
                 }
-                // There's no note ID, and/or it's assigned to the patient.
-                // In these cases a new note is created.
-                addPnote($reply_to, $note, $userauthorized, '1', $form_note_type, $assigned_to, '', $form_message_status);
+            } elseif ($_REQUEST['go']=="addRecall") {
+                $MedEx->display->display_add_recall();
+            } else if ($_REQUEST['go']=='Recalls') {
+                $MedEx->display->display_recalls($logged_in);
+            } elseif ((($_REQUEST['go']=="setup")||($_REQUEST['go']=='Preferences'))&&($logged_in)) {
+                $MedEx->display->preferences();
+            } elseif ($_REQUEST['go']=='icons') {
+                $MedEx->display->icon_template();
+            } elseif ($_REQUEST['go']=='SMS_bot') {
+                $MedEx->display->SMS_bot($logged_in);
+            } else {
+                echo xlt('Warning:  Navigation error.  Please behave.');
             }
-        }
-    } break;
-    case "savePatient":
-    case "save" : {
-        // Update alert.
-        $noteid = $_POST['noteid'];
-        $form_message_status = $_POST['form_message_status'];
-        $reply_to = $_POST['reply_to'];
-        if ($task=="save")
-            updatePnoteMessageStatus($noteid,$form_message_status);
-        else
-            updatePnotePatient($noteid,$reply_to);
-        $task = "edit";
-        $note = $_POST['note'];
-        $title = $_POST['form_note_type'];
-        $reply_to = $_POST['reply_to'];
-    }
-    case "edit" : {
-        if ($noteid == "") {
-            $noteid = $_GET['noteid'];
-        }
-        // Update the message if it already exists; it's appended to an existing note in Patient Notes.
-        $result = getPnoteById($noteid);
-        if ($result) {
-            if ($title == ""){
-                $title = $result['title'];
-            }
-            $body = $result['body'];
-            if ($reply_to == ""){
-                $reply_to = $result['pid'];
+            ?>
+    <?php } else {
+      //original message.php stuff
+        ?>
+      <div class="row">
+        <div class="col-sm-2"></div>
```

Comment:
Don't use empty grid, instead do `col-sm-8 col-offset-sm-2` on the next div. Also, `center` should be `text-center`